spec/features/trainer_spec.rb

```diff
@@ -14,6 +14,19 @@
       expect(page).to have_content 'Ожидайте наступления даты пересмотра.'
     end
   end
+
+  describe 'card does not exist' do
+    before do
+      @user = create(:user)
+      visit trainer_path
+      login('test@test.com', '12345', 'Войти')
+    end
+
+    it 'rescue an error' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.